doc/userguide/file-extraction/file-extraction.rst

```diff
@@ -63,6 +63,15 @@ of the filename. For example, if the SHA256 hex string of an extracted
 file starts with "f9bc6d..." the file we be placed in the directory
 `filestore/f9`.
 
+The size of a file that can be stored depends on ``file-store.stream-depth``,
+if this value is reached a file can be truncated and not stored completely.
```

Comment:
f this value is reached **,** a file can be truncated and **might not be** stored completely